### ISSUE TITLE
Converting a png file to a pdf creates a blank page

### ISSUE THREAD

**Issue by ipfaze** [mothsART/machin#5]
### What happened?

When trying to convert a png file to a pdf file, the command below produces a blank pdf.
```bash
ls ~/Downloads/pineapple.png | ~/Projects/Rust/machin/target/debug/machmap -e pdf
```

### What did you expect to happen?

We expect to generate a pdf file containing the image like it is when you use the command to convert a jpg file to a pdf.

### Machin version

I'm using the current version available on the master branch, the last commit hash is [2c33926edda9f7471331f90aa9b4d499bea4aaf6](https://github.com/mothsART/machin/commit/2c33926edda9f7471331f90aa9b4d499bea4aaf6)

### Steps to reproduce it

1. Pull the master branch
2. Build project using `cargo build`
3. Download a png image
4. Try to convert it using `ls image_file.png | machmap -e pdf`

The result of those steps should produce a blank pdf.

**Comment by mothsART** [comment]
Can you bring me an original png file and the result ?
I'me not sure to reproduce.

**Comment by ipfaze** [comment]
Hi !

I'm actually using this [image](https://pngimg.com/uploads/pineapple/pineapple_PNG2756.png) to test this feature.
The command `ls pineapple.png | machmap -e pdf` produce this file : 
[pineapple.pdf](https://github.com/mothsART/machin/files/9375628/pineapple.pdf)

Which is blank for me.

If you can't reproduce it, maybe it's my pc configuration which is in fault.
For information, I'm using a Macbook Pro 16 with m1 pro (arm architecture) with MacOS Monterey 12.5

Feel free to reach me out if you have any questions 🙂

**Comment by mothsART** [comment]
bugfix here : https://github.com/mothsART/machin/pull/13

I solved this with several representative images.
I will focus on unit tests now.

Big thanks.

**Comment by ipfaze** [comment]
Hi @mothsART 

#13 fixes this issue and also #9.

Thank you so much for your work !
